### PR TITLE
made style changes and updated the props of daily quiz component

### PR DESCRIPTION
**Pull request by sirElvin**

### CODE REVIEW

File: nepalingo-web/src/components/DailyQuiz.tsx [hunk 1]
```diff
@@ -1,32 +1,46 @@
...
-  const navigate = useNavigate(); // Get the navigate function
+
+interface DailyQuizProps {
+  backgroundImageUrl: string;
+  quizYourselfText: string;
+  descriptionText: string;
+  buttonText: string;
+}
+
+const DailyQuiz: React.FC<DailyQuizProps> = ({
```
Comment (SansCaar) [inline nepalingo-web/src/components/DailyQuiz.tsx:12]: Rename it to something like Activity Card or something that is more generic

File: nepalingo-web/src/components/DailyQuiz.tsx [hunk 2]
```diff
@@ -1,32 +1,46 @@
...
   return (
     <div
-      className="daily-quiz-card bg-cover text-white p-8 rounded-lg"
-      style={{ backgroundImage: `url(${backgroundImageUrl})` }}
+      className="daily-quiz-card bg-cover text-white p-8 rounded-lg flex flex-col justify-end"
+      style={{
+        backgroundImage: `url(${backgroundImageUrl})`,
+        borderRadius: "0.5rem",
+        height: "220px", // Reduced height
+        paddingBottom: "20px", // Space at the bottom
```
Comment (SansCaar) [inline nepalingo-web/src/components/DailyQuiz.tsx:31]: donot use style here use tailwind instead

File: nepalingo-web/src/components/DailyQuiz.tsx [hunk 3]
```diff
@@ -1,32 +1,46 @@
...
 
   const handleStartQuizClick = () => {
-    // Redirect to /flashcard when the button is clicked
     navigate("/flashcard");
   };
 
   return (
     <div
-      className="daily-quiz-card bg-cover text-white p-8 rounded-lg"
-      style={{ backgroundImage: `url(${backgroundImageUrl})` }}
```
Comment (SansCaar) [inline nepalingo-web/src/components/DailyQuiz.tsx:17]: Donot directly use a background image add an image which covers the whole height and width and also add an overlay on top of it with some color like bg-black/90

**Review by SansCaar — COMMENTED** [review]
One another thing is the card should take all the space available. 
So try to give it a flex 1 and an aspect ratio of 16:9 or something like that so it'll expand whenerver there is space and we can define the height widht based on its container

File: nepalingo-web/src/pages/Home.tsx [hunk 4]
```diff
@@ -43,7 +43,12 @@ const Home: React.FC = () => {
           </div>
         </div>
         <div className="mb-5 pt-5">
-          <DailyQuiz />
+          <ActivityCard 
+            backgroundImageUrl="https://t4.ftcdn.net/jpg/06/28/27/29/360_F_628272989_krphWxzqARpTZ20wOBU0YeH2Ecf7iCtI.jpg"
```
Comment (christikaes) [inline nepalingo-web/src/pages/Home.tsx:47]: where did this image come from? can we download and put in our repo? that way if this server goes down the image doesn't break

**Review by christikaes — COMMENTED** [review]
small comment, looks good if Sanskar's comments are addressed

**Review by SansCaar — COMMENTED** [review]
Fine for now but we need to redo things here....the background is not showing up, ani the ui is not matching the figma.

**Review by binamkayastha — APPROVED** [review]

**Pull request merged**